viewer/jest.config.js

```diff
@@ -7,15 +7,22 @@ const path = require('path');
 process.env.VERSION = 'test';
 process.env.MIXPANEL_TOKEN = 'test';
 
+const { jsWithTsESM: spec } = require('ts-jest/presets');
+
+spec.transform['^.+\\.m?[tj]sx?$'][1].tsconfig = 'tsconfig.test.json';
```

Comment:
Maybe a comment on what this actually is trying to match against?

Reply:
Removed this config altogether.. Seems like it was unnecessary all along